envoy/http/filter.h

```diff
@@ -626,6 +641,33 @@ class StreamDecoderFilterCallbacks : public virtual StreamFilterCallbacks {
   virtual absl::optional<absl::string_view> upstreamOverrideHost() const PURE;
 };
 
+/**
+ * Hepler used to set and get stream filter names. This is to avoid having to implement
+ * these method for every filter.
+ */
+class StreamFilterNameHepler {
```

Comment:
```suggestion
class StreamFilterNameHelper {
```

s/Hepler/Helper here and elsewhere.